parlai/mturk/core/dev/test/test_full_system.py

```diff
@@ -343,6 +342,9 @@ class TestMTurkManagerWorkflows(unittest.TestCase):
     """
 
     def setUp(self):
+        patcher = mock.patch('builtins.input', return_value='y')
```

Comment:
@JackUrb  this party you review.

Note that input was being done a nasty way before: we would set the context here, and that would cause calls to input later to resolve to a nonlocal context, instead of the builtins context. Very gross.

Now we'd like to just use the supported monkeypatching. If this weren't all being done in setUp, we could just use a decorator, but that doesn't affect setUp. So we need to manually patch.

Reply:
This is much cleaner than the silly solution I stumbled on. Looks good to me!